src/main/resources/log4j.properties

```diff
@@ -0,0 +1,64 @@
+# Root logger option
+log4j.rootLogger=INFO, stdout, file
+# Redirect log messages to console
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} %-5p %c{1}:%L - %m%n
+# Redirect log messages to a log file, support file rolling.
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=C:\\Users\\Alexandr\\IdeaProjects\\multithreading\\src\\main\\resources\\log4j.properties
```

Comment:
you should add logs to another file, for example "C:\\Users\\Alexandr\\IdeaProjects\\multithreading\\log\\logInform.log

Reply:
Thnx, changed